custom/icds_reports/utils.py

```diff
@@ -243,6 +243,30 @@ def apply_exclude(domain, queryset):
     )
 
 
+def get_age_filter(age_value):
+    """
+        When age_value comes from age filter it means first range is chosen 0-6 months.
+        For that range we want to include 0 and 6 in results.
+    """
+    if age_value == '6':
```

Comment:
can you explain this part a little more, im not sure i understand what is going on here

Reply:
We have a bunch of values in filter
0 - 6 months, 6 - 12 months, etc.
currently, age_tranche can have a values 0, 6, 12, 18 etc.
To not lose value '0' we need to include it in the first range 0-6. For the rest of ranges, we include only rightmost value.

0 -6 -> 0,6
6-12 -> 12
12-18 -> 18
etc.
As you can see without this clause we lose 0 value at all.